### ISSUE TITLE
IC2 - Gendustry Compatability Issue

### ISSUE THREAD

**Issue by quartermastersolar** [bdew-minecraft/gendustry#173]
Having an error on our server where it freezes the game due to a compatibility issue between IC2 and gendustry when registering the EU power type with the gendustry powered machines. The server doesn't crash but it ends up unresponsive with the console being spammed with the following exceptions:

[07:24:26] [Server thread/WARN] [IC2.EnergyNet]: EnergyNet.addTileEntity: net.bdew.gendustry.machines.apiary.TileApiary@2899aad is already added, aborting
[07:24:26] [Server thread/WARN] [IC2.EnergyNet]: EnergyNet.addTileEntity: net.bdew.gendustry.machines.apiary.TileApiary@478f0b4 is already added, aborting
[07:24:26] [Server thread/WARN] [IC2.EnergyNet]: EnergyNet.addTileEntity: net.bdew.gendustry.machines.apiary.TileApiary@5bb0cf1 is already added, aborting
[07:24:26] [Server thread/WARN] [IC2.EnergyNet]: EnergyNet.addTileEntity: net.bdew.gendustry.machines.apiary.TileApiary@48c0c7f is already added, aborting
[07:24:26] [Server thread/WARN] [IC2.EnergyNet]: EnergyNet.addTileEntity: net.bdew.gendustry.machines.apiary.TileApiary@75ebaa7 is already added, aborting
[07:24:26] [Server thread/WARN] [IC2.EnergyNet]: EnergyNet.addTileEntity: net.bdew.gendustry.machines.apiary.TileApiary@33362e2 is already added, aborting
[07:24:26] [Server thread/WARN] [IC2.EnergyNet]: EnergyNet.addTileEntity: net.bdew.gendustry.machines.apiary.TileApiary@1cdeae6 is already added, aborting
[07:24:26] [Server thread/WARN] [IC2.EnergyNet]: EnergyNet.addTileEntity: net.bdew.gendustry.machines.extractor.TileExtractor@52ef5d0 is already added, aborting

Using:

Gendustry Version: 1.6.4.11
Forestry: 5.2.1.204
Industrial Craft 2 Version: 2.6.1-ex110
Modpack: DonutCraft 1.10.2-0.0.6c on ATLauncher
Minecraft Forge: 12.18.1.2011
FML: 8.0.99.99

**Comment by bdew** [comment]
Weird, i'll look into it though in my test i didn't see any of that.

Is the area chunkloaded? Does it happen at some intervals or when doing something specific?

**Comment by quartermastersolar** [comment]
Hi bdew,

I have placed most of the gendustry machines down in four chunks with about 8 apiaries running. The chunks the machines are in are unloaded. The server seems to freeze up after 6hrs from a restart. We are using enderio conduits to power the machines and apiaries. We did further tests and found that chunk loading the area resolved the issue. Here are some pictures of the setup:
![2016-07-25_12 07 46](https://cloud.githubusercontent.com/assets/20620138/17089444/40319b08-526a-11e6-94f6-2f6a68171f25.png)
![2016-07-25_12 07 55](https://cloud.githubusercontent.com/assets/20620138/17089445/4033d53a-526a-11e6-98c0-7794096eddea.png)
![2016-07-25_12 08 04](https://cloud.githubusercontent.com/assets/20620138/17089443/4030095a-526a-11e6-94d7-0a055a8ccc41.png)
